packages/utils/src/account.ts

```diff
@@ -1,5 +1,9 @@
 import * as bip32 from 'bip32'
 import * as bip39 from 'bip39'
+// tslint:disable-next-line: no-duplicate-imports
+import { wordlists } from 'bip39' // Unify the wordlists (otherwise depends on the instance of the bip39)
```

Comment:
Why not access this through `bip39.wordlists` instead of creating a dupe import?

Reply:
Because I didn't had the wrapper before, hahaha. But you are totally right!